create_item_demo.py

```diff
@@ -0,0 +1,26 @@
+import requests
```

Comment:
These demo files are really useful when getting the feature set up & tested, but in the long term they're likely to be ignored and end up out of date, not quite matching how we actually solve the problem in the app, so I'm a big fan of deleting these files now! We can always recover them with Git later